app/views/groups/_form.html.erb

```diff
@@ -7,6 +7,10 @@
     <%= f.label :description %>
     <%= f.text_area :description %>
   </div>
+  <div>
+    <%= f.label :gift_due_date, 'Gift Due Date (YYYY/MM/DD)' %>
+    <%= f.text_field :gift_due_date %>
```

Comment:
there is a date field for rails form helper

Reply:
Feature added. (Pretty cool too!)

View logic added setting the minimum date allowed for entry.